modules/contracts/apps/challenge/examples/deploy.rs

```diff
@@ -0,0 +1,28 @@
+use cw_orch::{
+    anyhow,
+    prelude::{networks::parse_network, DaemonBuilder},
+    tokio::runtime::Runtime,
+};
+
+use abstract_challenge_app::{contract::CHALLENGE_APP_ID, ChallengeApp};
+use abstract_interface::AppDeployer;
+use semver::Version;
+
+const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+fn main() -> anyhow::Result<()> {
+    dotenv().ok();
+    env_logger::init();
+    let chain = parse_network("uni-6");
```

Comment:
Hm we should update the template to take in the network as the argument for this script.

Reply:
Made a ticket to track that here: https://linear.app/abstract-sdk/issue/ABS-192/update-template-to-in-network-as-arg